pkg/primitives/vm.go

```diff
@@ -282,6 +284,12 @@ func (p *Primitives) virtualMachineProvisionImpl(ctx context.Context, wl *gridty
 		if err := p.vmMounts(ctx, deployment, config.Mounts, true, &machine); err != nil {
 			return result, err
 		}
+		if config.Corex {
+			if err := p.copyFile("/usr/bin/corex", filepath.Join(mnt, "corex"), 0766); err != nil {
```

Comment:
I think the right permissions are 0755 which is rwx r-x r-x 6 is rw-. it doesn't matter as long as the root has execute permissions but just for the consistency,